msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -61,27 +64,31 @@ The following is a JSON example of an Adaptive Card-based Loop component with th
 
 ### Query parameters
 
-|Property|Type|Description|Required|
-|---|---|---|---|
-| `metadata`| Metadata | Defines various metadata properties typically not used for rendering the card. | No |
-| `webUrl` | String | URL that uniquely identifies the card and serves as a browser fallback that can be used by some hosts.| No |
+|Property|Type|Description|
+|---|---|---|
+| `metadata`| Metadata | Defines various metadata properties typically not used for rendering the card. |
+| `webUrl` | String | URL that uniquely identifies the card and serves as a browser fallback that can be used by some hosts. |
+
+> [!NOTE]
+> * When you copy an Adaptive Card-based Loop component, the URL of the Loop component is copied. When you paste the URL in a Teams chat or the Loop app, the link unfurls into the Adaptive Card-based Loop component.
+> * Adaptive Card-based Loop components use the same [refresh mechanism](/adaptive-cards/authoring-cards/universal-action-model#refresh-mechanism) as regular Adaptive Cards. The content in an Adaptive Card-based Loop component is refreshed only when a user opens the Loop component.
 
-## Test your Loop component
+## Test Adaptive Card-based Loop component
```

Comment:
Reading through this section again, I'm not sure it adds value to the article. It doesn't cover any test / debug guidance that's specific to this AC Loop component scenario (e.g., screenshots of sample app running in Teams/Outlook), and the steps aren't really a complete E2E of how to test your AC Loop component ME running in those hosts. Should we remove this section for now, and just summarize next steps (package, sideload) and say that from there you test your AC LC the same as you would run a regular ME?